lib/actions-oclif/envs.ts

```diff
@@ -241,20 +282,24 @@ async function getAppVars(
  * Precondition: options.device must be defined.
  */
 async function getDeviceVars(
-	sdk: import('balena-sdk').BalenaSDK,
+	sdk: SDK.BalenaSDK,
+	fullUUID: string,
+	appName: string | undefined,
 	options: FlagsDef,
 ): Promise<EnvironmentVariableInfo[]> {
-	const deviceUUID = options.device!;
+	const printedUUID = options.json ? fullUUID : options.device!;
```

Comment:
I've found that not all SDK methods accept short 7-char UUIDs (instead requiring the full 32-char UUID). So I thought that, when the user chooses JSON output (which is meant for programmatic use), the full UUID should always be printed instead of the short UUID. However in tabular format (which is meant for human consumption, and "visual neatness" is a little bit more of a consideration), the same UUID given by the user in the command line (`options.device`) is used -- if the user provided a short UUID, a short UUID is printed; if the user provided a full UUID, a full UUID is printed.